dojo/tools/spotbugs/parser.py

```diff
@@ -58,7 +67,9 @@ def __init__(self, filename, test):
                     active=False,
                     verified=False,
                     numerical_severity=Finding.get_numerical_severity(severity),
-                    static_finding=True
+                    static_finding=True,
+                    line=source_line,
+                    file_path=source_file
```

Comment:
Please add also these ones if it's relevant:
* sast_source_object
* sast_sink_object
* sast_source_line (int)
* sast_source_file_path

Reply:
Wha't the difference between : 

> * sast_source_line (int)

and line 

> * sast_source_file_path

and file_path

Reply:
same data